src/pocketmine/scheduler/AsyncTask.php

```diff
@@ -27,7 +27,7 @@
 /**
  * Class used to run async tasks in other threads.
  *
- * WARNING: Do not call PocketMine-MP API methods, or save objects from/on other Threads!!
+ * WARNING: Do not call PocketMine-MP API methods, or save objects (and arrays cotaining objects) from/on other Threads!!
```

Comment:
@remotevase Please immediately **stop** using pull requests and issues as chatroom or forums, or respective action will be taken.